cmd/kubeapps-apis/proto/kubeappsapis/plugins/helm/packages/v1alpha1/helm.proto

```diff
@@ -195,6 +195,18 @@ message HelmPackageRepositoryCustomDetail {
 
   // whether to perform validation on the repository
   bool perform_validation = 4;
+
+  // TODO
+  ProxyOptions proxy_options = 5;
+
+  // TODO
+  map<string, string> node_selector = 6;
+
+  // TODO
+  repeated Toleration tolerations = 7;
+
+  // TODO
+  PodSecurityContext security_context = 8;
```

Comment:
New options for the Helm plugin (repos API)
[will add the description soon, I just wanted to share my approach with you ASAP]